server/graphql/v1/types.js

```diff
@@ -241,6 +242,12 @@ export const UserType = new GraphQLObjectType({
           return user.data && user.data.features && user.data.features.ALL === false;
         },
       },
+      hasSeenLatestChangelogEntry: {
+        type: GraphQLBoolean,
+        async resolve(user) {
+          return await hasSeenLatestChangelogEntry(user);
```

Comment:
Any reason for adding this?

```suggestion
          return hasSeenLatestChangelogEntry(user);
```

Reply:
Ah yes, I think I was testing something; this isn't needed. 🤔